### DISCUSSION TITLE
Custom names in observation list?

### DISCUSSION THREAD

**Discussion by GerritErdt** [Stellarium/stellarium#3097] [General]
Hi guys, 

I plan my photography projects using the observation list feature. That is much more convenient than entering coordinates/object numbers every time. 

Is there a way to give a custom name to an object in the list? This would make it much easier to remember which object is which shot. Currently, I have to remember the SAO numbers of the stars I center on. 

CS Gerrit

**Comment by gzotti** [comment]
No, this is currently not possible. Object names are even reloaded on every start. Somebody would have to introduce a user-editable field.

**Reply by GerritErdt** [reply]
Allright, thanks for the confirmation.
